jpa/org.eclipse.persistence.jpa/src/main/java/org/eclipse/persistence/jpa/JpaEntityManagerFactory.java

```diff
@@ -29,7 +30,7 @@
  * </p>
  * @see jakarta.persistence.EntityManagerFactory
  */
-public interface JpaEntityManagerFactory {
+public interface JpaEntityManagerFactory extends EntityManagerFactory, AutoCloseable {
```

Comment:
I think, that `AutoCloseable` will be applied/moved later to `jakarta.persistence.EntityManagerFactory` as a part of JPA specification change?